Ansible/group_vars/kenet.yml

```diff
@@ -0,0 +1,161 @@
+---
+#My var just to be sure ansible is calling this file
+ronald: 1970233
+
+#Initialize generic with a value causing undefined in Ansible/roles/yaim/tasks/config_dirs.yml
+generic: generic
+
+#SkipCRL
+SkipCRLs: 'true'
+
+emi_service: UI
+needs_mysql: false
+yaim_options: '-n UI'
+yaim_path: /opt/glite/yaim
+yum_args: '--exclude="voms-clients-cpp"'
+bdii_port: 2170
+vo_sw_dir: /opt/exp_soft
+cvmfs_repos:
```

Comment:
You can't have known this,  but the apprepo and devrepo are not in production yet. removed these two and use only `fastrepo.sagrid.ac.za`